README.md

```diff
@@ -19,7 +20,44 @@ How to use:
 - Create or open a project
 - Run menu Example > Example Action, it should display a list of classes
 
+## Install UML Metamodel to Eclipse 
+
+Viatra engine requires the UML Metamodel to be able to define queries and execute transformations. Hence it is necessary to install UML Metamodel to Eclipse if someone would like to use/extend the _MD_plugin_skeleton_ project. On this page, we propose a generic workflow to install UML Metamodel as a _p2 repository_.
+
+### 1 Import MagicDraw's example project to Eclipse and change the target platform
+ 1. Start Eclipse
+ 1. File > Import > Existing project into workspace > Select archive file > `$Your MagicDraw installation$/openapi/ide/eclipse.zip` > check only `MagicDraw (MagicDraw/)` > click on "Finish".
+ 1. In the "Package Explorer", right click on `MAGIC_DRAW_INSTALL_DIRECTORY` under the newly imported project > Select "Properties" > Click on "Edit" > Set the value to `$Your MagicDraw installation$` > Click on "Ok".
+ 1. Open "MagicDraw bundles.target" > Click on "Set as Active Target Platform".
+
+### 2 Create a feature package to host a p2 repository of UML Metamodel
+ 1. File > New > Feature Project
+	* Project Name: `com.magicdraw.uml.metamodel`
+	* Feature ID: `com.magicdraw.uml.metamodel`
+	* Feature Name: `UML Metamodel`
+	* Feature Version: 1.0.0.*
+ 1. Click on "Next" to select the referenced plugins
+ 1. Select "com.nomagic.magicdraw.uml2" and its dependencies (check section **Dependencies of UML2 plugin** at the [wiki page](https://github.com/IncQueryLabs/MD_plugin_skeleton/wiki/Install-UML-Metamodel-to-Eclipse))
```

Comment:
I'd rather avoid this process defined twice: once in the readme and once in the wiki page. We should have only a single process description, otherwise they will get out-of-synch.